htdocs/product/price.php

```diff
@@ -141,11 +141,12 @@
 			// We look into database using code (we can't use get_localtax() because it depends on buyer that is not known). Same in create product.
 			$vatratecode = $reg[1];
 			// Get record from code
-			$sql = "SELECT t.rowid, t.code, t.recuperableonly, t.localtax1, t.localtax2, t.localtax1_type, t.localtax2_type";
+			$sql = "SELECT t.rowid, t.type_vat, t.code, t.recuperableonly, t.localtax1, t.localtax2, t.localtax1_type, t.localtax2_type";
 			$sql .= " FROM ".MAIN_DB_PREFIX."c_tva as t, ".MAIN_DB_PREFIX."c_country as c";
 			$sql .= " WHERE t.fk_pays = c.rowid AND c.code = '".$db->escape($mysoc->country_code)."'";
 			$sql .= " AND t.taux = ".((float) $tva_tx)." AND t.active = 1";
 			$sql .= " AND t.code = '".$db->escape($vatratecode)."'";
+			$sql .= " AND t.type_vat = 1";
```

Comment:
Why this test ?
We alreayd have the rate+code that make the vat rate unique.

Reply:
Because we only want the type 1 VAT rates, i.e. the sales type VAT rates.

I can remove but we are on the page of customer prices so we only display the VAT rates of the sales type, it is useless to display those of the purchases.

You sell at 20%, you don't want the 10%, 5.5%, 2.1% rates to be displayed.

Reply:
@eldy I proposed to display type 0 (VAT All type) + type 1 (VAT Sales type)